packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -303,11 +303,33 @@ const createFlowsheetResultCellDisplay = (resultDataSet, hideUnit, numericOverfl
     });
     const displayCount = additionalResultCount + 1;
     const hasAccessoryIcons = (firstResultAttributes.comment || firstResultAttributes.modified || firstResultAttributes.unverified);
-    const additionalResultsStackDisplay = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
+    const additionalResultsStack = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
+
+    if (hasAccessoryIcons) {
+      const endAccessoryIcons = createEndIcons(firstResultAttributes.comment, firstResultAttributes.modified, firstResultAttributes.unverified, resultKeyID);
+      const endDisplay = (
+        <div key="EndDisplay-AdditionalResultsAndIcons" className={cx('end-display')}>
+          {additionalResultsStack}
+          {endAccessoryIcons}
+        </div>
+      );
+      compositeCell.push(endDisplay);
+      return compositeCell;
+    }
+    const additionalResultsStackDisplay = (
+      <div key="EndDisplay-AdditionalResults" className={cx('end-display')}>
+        {additionalResultsStack}
+      </div>
+    );
     compositeCell.push(additionalResultsStackDisplay);
   }
   const endAccessoryIcons = createEndIcons(firstResultAttributes.comment, firstResultAttributes.modified, firstResultAttributes.unverified, resultKeyID);
-  compositeCell.push(endAccessoryIcons);
+  const endAccessoryIconsDisplay = (
+    <div key="EndDisplay-Icons" className={cx('end-display')}>
+      {endAccessoryIcons}
+    </div>
+  );
+  compositeCell.push(endAccessoryIconsDisplay);
```

Comment:
Are these changes specifically for the fix of additional results squares and end accessory icons? I pulled the branch to my local and run the site. Looks like squares and icons are showing up on the right side. Is this where we are right now?
<img width="1642" alt="Screenshot 2023-06-08 at 3 40 28 PM" src="https://github.com/cerner/terra-clinical/assets/44185348/9af1f066-356c-435c-8378-f58e9059d2db">

Reply:
This is a little complicated but here was my thought process:

When trying to format for a semantic table the spacing was off (you can see in the screenshot in the top description). After making changes to get them all on the same line those squares and icons still needed to be moved to the right, so I tried to use [float](https://www.w3schools.com/cssref/pr_class_float.php) on the them.

The way the logic in this file worked before was if there were additional results then the squares were created and pushed first, then if there were icons those were created and pushed. So visually if both were present then both would be on the right hand side of the cell with the squares coming first and the icons to the right of the squares.

The tricky part is when trying to use float it picks up whichever element was added first (squares) and moves it as far right as it can go, then the next one is added (icons) and is moved right until it hits the first element. So basically they were flipped in the wrong order, icons on the left and squares on the right. Saw some suggestions to just swap the order of the elements but because the rest of the UI currently uses non-semantic elements like divs - if we try to swap them it messes up that non-semantic formatting. 

My solution was to wrap the squares and the icons in a parent container so I could just float the entire container to the right while still maintaining the proper order of the squares and icons. I checked the rest of the tests and site examples and everything still looks the same, no wdio errors so the non-semantic formatting hasn't changed.

Reply:
Also I'm using the parent container around the elements even if they are alone. This is so we don't have to worry about floating those individually when the other isn't there, they'll just always be in the container to the right. Otherwise when we only have the squares or only have the icons we'll continue to have the issue where they are on the left with the text.